utils/host/src/block_range.rs

```diff
@@ -42,3 +44,17 @@ pub async fn get_validated_block_range(
 
     Ok((l2_start_block, l2_end_block))
 }
+
+/// Get a fixed recent (less than 2 weeks) block range.
+pub async fn get_rolling_block_range(
+    data_fetcher: &OPSuccinctDataFetcher,
+    range: u64,
+) -> Result<(u64, u64)> {
+    let now = SystemTime::now().duration_since(UNIX_EPOCH)?.as_secs();
+    let start_timestanp = now - (now % (14 * 24 * 60 * 60));
```

Comment:
nit: Pass the "fixed" window (2 weeks) to the function, and use a constant for it inside of the multi/cost estimator script.